domain/src/main/java/edu/nextstep/camp/calculator/domain/Calculator.kt

```diff
@@ -3,17 +3,21 @@ package edu.nextstep.camp.calculator.domain
 class Calculator {
     fun evaluate(expression: String): Int {
```

Comment:
```suggestion
    fun evaluate(expression: Expression): Int {
```
Expression 클래스를 활용해보면 어떨까요?